tools/render-cloudcfg

```diff
@@ -4,6 +4,9 @@ import os
 import sys
 import argparse
 
+VARIANTS = ["almalinux", "alpine", "amazon", "arch", "centos", "cloudlinux", "debian",
```

Comment:
This list changed locations on the main branch since your PR was opened. You'll need to update the list below rather than the change here.

Reply:
Thank you for the review @holmanb . I have updated the file. Thanks.